deprecations/README.md

```diff
@@ -39,6 +39,7 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| Extent `RateGroups`<br>in [Get all rates](../operations/rates.md#get-all-rates) | Use [Get all rate groups](../operations/rategroups.md#get-all-rate-groups) instead | 30 Dec 2024 | 30 Dec 2025 |
```

Comment:
suggestion: The deprecation day still corresponds to the date when we announce the deprecation, it's the discontinuation date which is shifting.

Per discussion with Sylvia, we have moved the batched discontinuations to January 10th since killing stuff on December 30th seems pretty harsh (and realistically, we wouldn't really do it). (Sorry we didn't announce it, it's not an official policy yet.)

```suggestion
| Extent `RateGroups`<br>in [Get all rates](../operations/rates.md#get-all-rates) | Use [Get all rate groups](../operations/rategroups.md#get-all-rate-groups) instead | 2 Feb 2024 | 10 Jan 2026 |
```